data/sql/updates/pending_db_world/rev_1662478777167897100.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `quest_template_addon` ADD COLUMN `ScriptName` VARCHAR(64) DEFAULT '' NOT NULL AFTER `TimeAllowed`;
```

Comment:
@SargeroDeV Please remove the empty line

Reply:
Sorry, in that sql I only see 1 line what do you mean by empty line?

Reply:
at the final of the file, you put an End of Line empty.